src/history/history-detail-overview.component.tsx

```diff
@@ -39,49 +39,46 @@ const HistoryDetailOverview: React.FC<HistoryDetailOverviewProps> = () => {
   const [ips, setIps] = useState({
     history: null,
     isLoading: true,
-    error: "",
-  })
-  
+    error: '',
+  });
+
   useEffect(() => {
     if (history != null) {
-      setIps((ps) => ({...ps, history: history, isLoading: isLoading, error: error?.message}));
+      setIps((ps) => ({ ...ps, history: history, isLoading: isLoading, error: error?.message }));
     }
-  },[error, history, isLoading])
-
+  }, [error, history, isLoading]);
 
   const handleSubmit = () => {
     setIsSubmitting(true);
-    createIpsResource(
-      uuid, 
-      abortController,
-    ).then((response) => {
-      if (response.status === 200) {
-        setIps((ps) => ({...ps, history: history, isLoading: isLoading, error: error?.message}));
+    createIpsResource(uuid, abortController)
+      .then((response) => {
+        if (response.status === 200) {
```

Comment:
What would happen if the response is not 200? I think we'd be better served by removing this check.